libs/tx-etransfer/src/transitions/sendETransfer.ts

```diff
@@ -71,17 +71,3 @@ function decryptInstructions(packet: EncryptedPacket) {
   return decryptTo<ETransferPacket>(privateKey, packet);
 }
 
-//
```

Comment:
This only ran on the server.  Moved into common code so we can check prior to submit *just in case*.  It would have caught several issues missed client-side recently

Reply:
Always better to double check 👍🏻